Modules/Core/Mesh/test/itkTriangleMeshCurvatureCalculatorTest.cxx

```diff
@@ -0,0 +1,144 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#include <iostream>
+
+#include "itkRegularSphereMeshSource.h"
+#include "itkTriangleMeshCurvatureCalculator.h"
+#include "itkTestingMacros.h"
+#include "itkMeshFileReader.h"
+
+int
+itkTriangleMeshCurvatureCalculatorTest(int argc, char * argv[])
+{
+  // Declare the type of the input mesh
+  using TriangleMeshType = itk::Mesh<double, 3>;
```

Comment:
I'd declare the mesh dimension as a constant so that it can be re-used in the few places you use it further below.

And even if the point type is not used, I'd also declare it as variable for the sake of consistency.